src/components/Cards/Available.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import ResourceAreaChart from '../Charts/AreaChart';
+
+const card = (
+    <React.Fragment>
+      <CardContent>
+        <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
```

Comment:
hard coding the styles is not the best practice, in case we need to support different themes and make it responsive to all different types of display